eng/pipelines/repoman.yml

```diff
@@ -107,6 +91,12 @@ jobs:
               )
             )
           )
+        inputs:
+          pwsh: true
+          targetType: filePath
+          filePath: ./eng/scripts/Write-CommentFile.ps1
```

Comment:
Thanks! We actually reviewed the docs and elected to go with Write as it's on the approved verbs a Communication Verb. The thinking was that the purpose of this script is to create the markdown file with the comment text, so that is the target. 

 https://docs.microsoft.com/en-us/powershell/scripting/developer/cmdlet/approved-verbs-for-windows-powershell-commands?view=powershell-7.2#communications-verbs